gocd/api/stage.py

```diff
@@ -0,0 +1,71 @@
+from gocd.api.endpoint import Endpoint
+
+__all__ = ['Stage']
+
+
+class Stage(Endpoint):
+    base_path = 'go/api/stages/{id}'
+
+    def __init__(self, server, pipeline_name, pipeline_counter, stage_name):
```

Comment:
This is my suggested change in a commit: https://github.com/gaqzi/py-gocd/commit/4dfd4c26c76cc639b51ef13863d557ba22647f20